sysinit/coqloadpath.ml

```diff
@@ -69,7 +54,7 @@ let init_load_path ~coqenv =
   let misc_ml, misc_vo =
     List.map (fun s -> build_userlib_path ~unix_path:s) (xdg_dirs @ rocqpath) |> List.split in
 
-  let ml_loadpath = plugins_dirs @ meta_dir @ contrib_ml @ List.concat misc_ml in
+  let ml_loadpath = meta_dir @ contrib_ml @ List.concat misc_ml in
```

Comment:
@ejgallego do you think it would also make sense to remove contrib_ml (user-contrib) and misc_ml (xdg dirs + COQPATH)?
COQPATH at least is only documented for the vo side AFAICT

Reply:
Yes I do, good catch, and thanks for these cleanups.

As far as I know metacoq is the most complex project that may still depend on this somehow. So the answer is yes, it is welcome, and IMHO if it doesn't break the metacoq developer build things should go fine.

Reply:
https://github.com/coq/coq/pull/19842